packages/astro-reactive-form/index.ts

```diff
@@ -1,3 +1,5 @@
 import Form from './Form.astro';
 export default Form;
+export * from './core/form-control-types';
+export * from './client';
```

Comment:
I think these two lines can be removed. We don't need to expose all form-control-types outside the package for now, and the `client` is already exported via package.json